lib/tasks/fake_data/journeys.rb

```diff
@@ -79,40 +79,15 @@ def create_journey(timestamp, journey_from, journey_to, billable)
         )
       end
 
-      def create_lockout_journey_event(journey, timestamp, notes, location)
-        journey.events.create!(
-          event_name: 'lockout',
-          client_timestamp: timestamp,
-          details: {
-            fake: true,
-            supplier_id: supplier.id,
-            event_params: {
-              attributes: {
-                notes: notes,
-              },
-              relationships: {
-                from_location: { data: { id: location.id } },
-              },
-            },
-          },
-        )
-      end
-
       def create_redirect_move_event(timestamp, notes, location)
-        move.move_events.create!(
```

Comment:
TODO: If we're not moving to v2 events in the diagnostics endpoint we should just dereference move_events here and preserve creation of v1 events until we migrate away from dual writing.